packages/react-experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/SelectedPersona.tsx

```diff
@@ -180,12 +178,7 @@ const SelectedPersonaInner = React.memo(
             className={css('ms-PickerItem-content', classNames.itemContentWrapper)}
             id={'selectedItemPersona-' + itemId}
           >
-            <Persona
-              {...item}
-              size={PersonaSize.size32}
-              styles={classNames.subComponentStyles.personaStyles}
-              coinProps={coinProps}
-            />
+            <Persona {...item} styles={classNames.subComponentStyles.personaStyles} />
```

Comment:
as size is an optional property, I do think we should set it to size32 if the consumer has not already set it

Reply:
What is the default size supposed to be? A comment on the PersonaSize enum seems to say it's supposed to be 48:

```
    /**
     * `regular` size has been deprecated in favor of standardized numeric sizing. Use `size48` instead.
     * @deprecated Use `size48` instead.
     */
    regular = 4,
```


Also, when I don't specify a size, it **does** default to 48.

Reply:
I meant that if there isn't one specified, we should use PersonaSize.size32 as that's what it was previously for this specific control

Reply:
Updated the default in a way that I think is reasonable, but I'm also unfamiliar with conventions used in JavaScript, so feel free to nitpick and make suggestions :)